agent/app/agent_capability_windows_test.go

```diff
@@ -183,7 +183,9 @@ func TestSupportedCapabilitiesWindows(t *testing.T) {
 		attributePrefix + capabilityContainerOrdering,
 		attributePrefix + capabilityFullTaskSync,
 		attributePrefix + capabilityEnvFilesS3,
-		attributePrefix + taskENIBlockInstanceMetadataAttributeSuffix}
+		attributePrefix + taskENIBlockInstanceMetadataAttributeSuffix,
+		attributePrefix + capabilityContainerIAMRole,
```

Comment:
I thought we don't support this feature on windows?

Reply:
container iam roles are generic and are unreleated to firelens v2, so I thought it would be appropriate to add it windows too!

Reply:
make sense